### PR TITLE
Edit description about this repo

### PR DESCRIPTION
**Pull request by sguinetti**
This PR is intended to better describe in a nutshell like a "Wikipedia" basedata repository of IPTV stations. Do you agree with this?

### CODE REVIEW

**Comment by Dum4G** [comment]
Unfortunately not. It should be easy to understand for people with A2 Pre-Intermediate level of speech and describe what people get and what they don't from this repo

File: .readme/template.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Collection of publicly available IPTV channels from all over the world.
 
-Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks.
+Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks. This repository has a community database for compile into organized and ready-to-use playlists the available IPTV stations, regardless of their local or international scope.
```
Comment (LaneSh4d0w) [inline .readme/template.md:7]:
```suggestion
Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks. This repository has a community database that compiles into organized and ready-to-use playlists the available IPTV stations, regardless of their local or international scope.
```

**Comment by LaneSh4d0w** [comment]
> Unfortunately not. It should be easy to understand for people with A2 Pre-Intermediate level of speech and describe what people get and what they don't from this repo

In this way, we could modify his sentence to : 

"This repository is maintained by a community that organizes into different playlists all the available IPTV stations, whether they are local or international."

**Comment by dct-infra** [comment]
> > Unfortunately not. It should be easy to understand for people with A2 Pre-Intermediate level of speech and describe what people get and what they don't from this repo
> 
> In this way, we could modify his sentence to :
> 
> "This repository is maintained by a community that organizes into different playlists all the available IPTV stations, whether they are local or international."

all the available **FREE**/**FREELY ACESSIBLE AVAILABLE** etc IPTV stations ? Insist on the free stuff.

File: .readme/template.md (hunk 1 above)
Comment (sguinetti) [reply .readme/template.md]: I changed the sentence to one you suggested @LaneSh4d0w.

**Comment by sguinetti** [comment]
> > > Unfortunately not. It should be easy to understand for people with A2 Pre-Intermediate level of speech and describe what people get and what they don't from this repo
> > 
> > 
> > In this way, we could modify his sentence to :
> > "This repository is maintained by a community that organizes into different playlists all the available IPTV stations, whether they are local or international."
> 
> all the available **FREE**/**FREELY ACCESSIBLE** etc IPTV stations ? Insist on the free stuff.

I prefer "open access", like "open access papers" term. Calling the sentence with "free" can generate confusion like "free speech" or "free license".

File: .readme/template.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Collection of publicly available IPTV channels from all over the world.
 
-Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks. This repository is maintained by a community that organizes into different playlists all the available IPTV stations without restrictions of access, whether they are local or international. Address of IPTV stations are obtained from official websites or authorized services.
+Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks. This repository is maintained by a community that organizes into different playlists all the available IPTV stations without restrictions of access, whether they are local or international. IPTV stations have  address, them are obtained from official websites or authorized services for work fastly in any media player with Internet connection.
```
Comment (LaneSh4d0w) [inline .readme/template.md]: Bit of oopsies here and there.

IPTV stations have addresses that are obtained [...]

[...] or authorized services for easier access for the end user that has any media player that supports streaming.

Comment (sguinetti) [reply .readme/template.md]: I just edited the sentence and added "web addresses" to the word "addresses".

**Comment by sguinetti** [comment]
I updated the text to add a paragraph to link Resources for those people new to the subject.

**Comment by LaneSh4d0w** [comment]
Here's my take on it.

**Comment by sguinetti** [comment]
I believe this PR is ready for review.

**Comment by freearhey** [comment]
For me personally, this repository is just a list of publicly available IPTV channels from around the world supported by the community and as a user I only open the description to copy the link to the playlist. For those who want to make a contribution there is a Contributing Guide. For rights holders there is a Legal section. So it's just not clear to me who this text is intended for?

**Comment by sguinetti** [comment]
The text is explained, in a friendly and summarized way, for those non-geeks who never understood an online TV. The text subtly condenses how IPTV works versus traditional TV. There are people who do not know that functions go beyond a simple movie player (think of those who do not live in developed countries for example that do not care to use VLC or other). In addition, this text encourages TV owners to introduce their own signals into the list (by the way, I don't think owners use Github and will need help from others to incorporate to their list).

**Comment by freearhey** [comment]
Thank you for clarifying. But the point is that the people you are describing will never read this text, because they simply do not know about the existence of this repository. 

Maybe it would be better to just edit the IPTV article on Wikipedia.

**Comment by sguinetti** [comment]
> [...] because they simply do not know about the existence of this repository.

I know not everyone will read it, but someone will check about this repo for curiosity. Anyway I will reduce the sentences to make it at least understandable (I already explained that the description should be friendly and it is better for IPTV newbies).

**Comment by freearhey** [comment]
> is there any way to add to the channel other than via Github

No, there is no such option.

**Comment by LaneSh4d0w** [comment]
For this PR, what do we suggest ? @Dum4G @freearhey

**Comment by freearhey** [comment]
I've already made my point, but if others think these edits are necessary, then PR should certainly be accepted.

**Comment by sguinetti** [comment]
I moved to the About section to explain how the community manages this "database" of TV stations in the simplest possible way and to explain its purpose for those who do not know much about technology (also accompanying the legal and contributing).

**Review by AnsahMohammad — APPROVED** [review]

**Review by AnsahMohammad — CHANGES_REQUESTED** [review]
The line 133:
Also, you will need fast connection for stream some channels in high resolution (720p, 1080p, 2160p or more). This means that if you have a slow connection, you might experience stutters and/or signal loss.

doesn't really fit under the header Resources

**Comment by sguinetti** [comment]
As the review software checks the available resolution, it fits well in the About section. I will move in the description to make the idea more condensed.

**Pull request closed without merging**